### PR TITLE
[W5.7][W14-1]Arsalan Cheema

### PR DESCRIPTION
**Pull request by arsalan0c**
Ready for review

### CODE REVIEW

**Comment by jjlee050** [comment]
Nice to add a new feature outside of the suggested enhancement. Nice PR.

**Review by jackedelic — COMMENTED** [review]
Generally good. Two thumbs up.

File: src/seedu/addressbook/commands/SizeCommand.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package seedu.addressbook.commands;
+
+/**
+ * Displays the total number of addresses.
```
Comment (nhs-work) [inline src/seedu/addressbook/commands/SizeCommand.java:4]: Comment does not match the explanation of the command.

File: src/seedu/addressbook/parser/Parser.java [hunk 2]
```diff
@@ -11,17 +11,7 @@
...
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```
Comment (nhs-work) [inline src/seedu/addressbook/parser/Parser.java:14]: Avoid importing using wildcards.

**Review by nhs-work — APPROVED** [review]
Good job on this week's PR! 👍 

Do note that while you have updated the IO tests, you did not update the JUnit tests for this new command.

You can close this PR once you have read my comments :)

**Pull request closed without merging**